scrapi/base/helpers.py

```diff
@@ -149,3 +152,21 @@ def language_code(language):
         return languages.get(name=language)
     except KeyError:
         return None
+
+
+def get_records_and_token(url, throttle, force, namespaces):
```

Comment:
Might want to namespace this as OAI, since it will only work for oai. We could also move all the OAI stuff to its own file, which wouldn't be a terrible idea at this point... Imports would change from ```from scrapi.base import <x>``` to ```from scrapi.base.oai import <x>```. If you don't want to make this change now, understandable.